doc/source/management/data_at_rest_encryption.rst

```diff
@@ -1,804 +1,11 @@
-are encrypted as well. The key which was used to encrypt the InnoDB system
-tablespace is also used to encrypt the doublewrite buffer.
-
-|Percona Server| encrypts the ``parallel doublewrite buffer`` with the respective
-tablespace keys. Only encrypted tablespace pages are written as encrypted in the
-parallel doublewrite buffer. Unencrypted tablespace pages will be written as
-unencrypted.
-
-.. important::
-
-   A server instance bootstrapped with the encrypted InnoDB system tablespace
-   cannot be downgraded. It is not possible to parse encrypted InnoDB system
-   tablespace pages in a version of |Percona Server| lower than the version
-   where the InnoDB system tablespace has been encrypted.
-
-
-System variables
---------------------------------------------------------------------------------
-
-.. variable:: innodb_sys_tablespace_encrypt
-
-   :cli: ``--innodb-sys-tablespace-encrypt``
-   :dyn: No
-   :scope: Global
-   :vartype: Boolean
-   :default: ``OFF``
-
-Enables the encryption of the InnoDB System tablespace. It is essential that the
-server is started with the ``--bootstrap`` option.
-
-.. seealso::
-
-   |MySQL| Documentation: ``--bootstrap`` option
-      https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_bootstrap
-
-.. variable:: innodb_parallel_dblwr_encrypt
-
-   :cli: ``--innodb-parallel-dblwr-encrypt``
-   :dyn: Yes
-   :scope: Global
-   :vartype: Boolean
-   :default: ``OFF``
-
-Enables the encryption of the parallel doublewrite buffer. For encryption, uses
-the key of the tablespace where the parallel doublewrite buffer is used.
-
-
-.. _innodb_general_tablespace_encryption:
-
-InnoDB General Tablespace Encryption
-================================================================================
-
-In |Percona Server| :rn:`5.7.20-18` existing tablespace encryption support has
-been extended to handle general tablespaces. A general tablespace is either
-fully encrypted, covering all the tables inside, or not encrypted.
-It is not possible to have encrypted only some of the tables in a general
-tablespace.
-
-This feature extends the  `CREATE TABLESPACE
-<https://dev.mysql.com/doc/refman/8.0/en/create-tablespace.html>`_
-statement to accept the ``ENCRYPTION='Y/N'`` option.
-
-.. note::
-
-   Prior to |Percona Server| 8.0.13, the ``ENCRYPTION`` option was specific to
-   the ``CREATE TABLE`` or ``SHOW CREATE TABLE`` statement.  In |Percona Server|
-   8.0.13, this option becomes a tablespace attribute and is not allowed with
-   the ``CREATE TABLE`` or ``SHOW CREATE TABLE`` statement except for
-   file-per-table tablespaces.
-
-   .. variable:: default_table_encryption
-
-       :cli: ``default-table-encryption``
-       :dyn: Yes
-       :scope: Session
-       :vartype: Text
-       :default: OFF
-
-Defines the default encryption setting for general tablespaces. The variable allows you to create or alter tables without specifying the ENCRYPTION clause.
```

Comment:
not only for general tablespaces, but also for SCHEMA.

Reply:
Will add schema.